src/api.ts

```diff
@@ -104,9 +108,15 @@ export async function removeDependency(
   const args = (
     resolvedOptions.packageManager.name === "yarn"
       ? [
+          resolvedOptions.global
+            ? resolvedOptions.packageManager.majorVersion === "1"
+              ? "dlx"
```

Comment:
What this line correct? 👀 It is like npx https://yarnpkg.com/cli/dlx

Reply:
https://github.com/yarnpkg/berry/issues/821#issuecomment-579372317

Reply:
It means it is "unsupported"/deprecated. DLX and global install are not the same. DLX runs the command when invoked. global install makes it available in path.

Reply:
https://github.com/unjs/nypm/commit/f5e4a39e1a8103b7cf8c7aeabbaea63ad5a30271